### ISSUE TITLE
Add ability to spawn asteroids

### ISSUE THREAD

**Issue by jrossignol** [jrossignol/ContractConfigurator#184]

**Comment by linuxgurugamer** [comment]
Just an FYI, there is already a mod for custom asteroids:  http://forum.kerbalspaceprogram.com/threads/80483

It's for 0.25, but seems to work on 0.90.  Might save you some work if you look at it.

**Comment by jrossignol** [comment]
Yup, aware of it - it was one of the examples I looked at when looking at how to spawn vessels. :+1:
